entry.go

```diff
@@ -13,6 +13,7 @@ import (
 )
 
 type Entry struct {
+	Version uint8    `json:"version"`
```

Comment:
The json marshalling is overridden in a custom MarshalJson function. The version is not included there, or in the unmarshal json